lib/iex/lib/iex/evaluator.ex

```diff
@@ -108,13 +127,17 @@ defmodule IEx.Evaluator do
     end
   end
 
-  defp adjust_operator([{op_type, _, _} | _] = tokens, line, column, file, opts)
+  defp adjust_operator([{op_type, _, token} | _] = _tokens, line, column, _file, _opts, :match)
+       when op_type in @op_tokens,
+       do: {:error, {[line: line, column: column], :malformed_pipe, "('#{token}')"}}
```

Comment:
There are no parens around the token in the error message!

```suggestion
       do: {:error, {[line: line, column: column], :malformed_pipe, "'#{token}'"}}
```